src/scripts/modules/media/body/content-baked.less

```diff
@@ -122,36 +143,83 @@ h1.example-title .text {
 
 // Helper mixin for expanding all the selectors
 .make-block(@type) {
+  #blockish>.style(@type);
```

Comment:
@philschatz that is not working either. the styles under `#blockish` in `content-baked.less` aren't being overridden.